<launch>

<!--
    Run as, e.g., 

    roslaunch nist_kitting kitting_demo_2.launch inifile_name:=/usr/local/iora/etc/kittingDemo.ini debug:=-d nocheck:=-X
-->

<arg name="inifile_name" default="ini.ini" />
<arg name="nocheck" default="" />
<arg name="debug" default="" />
<arg name="ws_all" default="" />

<group>

  <node pkg="nist_kitting" name="robot_prim" args="-i $(arg inifile_name)" type="crclsim.py" output="screen"/>

  <node pkg="nist_kitting" name="gripper_prim" args="-i $(arg inifile_name) -n gripper_prim" type="crclsim.py" output="screen"/>

  <node pkg="nist_kitting" name="emove" args="-i $(arg inifile_name) $(arg nocheck) $(arg debug)" type="kitting_emove.py" output="screen"/>

  <node pkg="nist_kitting" name="pl_hi" args="-i $(arg inifile_name)" type="kitting_pl_hi_node" output="screen"/>

  <node pkg="nist_kitting" name="hmi_proxy" args="-i $(arg inifile_name) $(arg debug) $(arg ws_all)" type="kitting_hmi_proxy" output="screen"/>

  <node pkg="nist_kitting" name="hmi" args="-i $(arg inifile_name)" type="kitting_hmi.py" output="screen" required="true"/>

</group>

</launch>
